swan-lake/learn-the-language/language-basics.md

```diff
@@ -227,9 +227,21 @@ int n = s.length();
 
 The ``substring()`` and ``length()`` functions are ``string`` lang library functions called using the convenient method call syntax. However, the functions are called on variables/values of the ``string`` type rather than objects.
 
-Ballerina imports the lang library for **``ballerina/lang.T``** where **``T``** represents a built-in type. Therefore, in the case of the above code example, you can also find the length of the string value by importing the **``ballerina/lang.string``** module (**``import ballerina/lang.'string``**) and calling the **``length()``** function using the function call syntax.
+A lang library **``ballerina/lang.T``** is defined for each built-in type **``T``**. Ballerina automatically imports these lang libraries and pre-declares a prefix **``T``**, which can be used to access constructs of the module. Therefore, in the case of the above code example, you can also find the length of the string value referring to the **``length()``** function as **``string:length()``** and using the function call syntax.
```

Comment:
```suggestion
A lang library **``ballerina/lang.T``** is defined for each built-in type **``T``**. Ballerina automatically imports these lang libraries and pre-declares a prefix **``T``**, which can be used to access the constructs of the module. Therefore, in the case of the above code example, you can also find the length of the string value referring to the **``length()``** function as **``string:length()``** using the function call syntax.
```

Reply:
Thanks, will change to
```suggestion
A lang library **``ballerina/lang.T``** is defined for each built-in type **``T``**. Ballerina automatically imports these lang libraries and pre-declares a prefix **``T``**, which you can use to access the constructs of the module. Therefore, in the case of the above code example, you can also find the length of the string value referring to the **``length()``** function as **``string:length()``** and using the function call syntax.
```

Reply:
Cool, also I think the end should also change
this > **``length()``** function as **``string:length()``** and using the function call syntax.
to > **``length()``** function as **``string:length()``** using the function call syntax.

Reply:
The guides seem to have used parentheses when referring to functions rather than function calls. Here also, we are showing how we are referring to it and then asking to use the function call syntax. So the current version is OK IMO, WDYT?

Reply:
Basically, if there wasn't a pattern to stick to, I would say

> Therefore, in the case of the above code example, you can also find the length of the string value referring to the **``length``** function as **``string:length``** and using the function call syntax.

Reply:
+1